src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -305,13 +308,15 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
               small
               nameWithUnderscore
               imgUrl={room.image_url}
+              text="Upload a new icon"
+              subtext="(A transparent 300 px square image works best)"
             />
             {errors?.room?.image_url && (
               <span className="input-error">
                 {errors?.room?.image_url.message}
               </span>
             )}
```

Comment:
Since `ImageInput` is being changes, maybe it was an opportunity lost to fold this inside it like in other `Admin*` input components.

Reply:
`ImageInput` currently has `error` prop, so I've used that in b6cf248 and removed unnecessary `span` elements with errors from render function. [NOTE] Need to modify the way errors are being passed/fetched to the `ImageInput` , so that it's the same with other `Admin*` components ( using lodash `get` etc. )